pkg/aggregator/demultiplexer.go

```diff
@@ -115,6 +119,10 @@ func createIterableMetrics(
 			if isServerless {
 				log.DebugfServerless("Sending sketches payload : %s", sketch.String())
 			}
+			hostTags := hostTagProvider.GetHostTags()
+			if hostTags != nil {
+				sketch.Tags = tagset.CombineCompositeTagsAndSlice(sketch.Tags, hostTagProvider.GetHostTags())
```

Comment:
```suggestion
				sketch.Tags = tagset.CombineCompositeTagsAndSlice(sketch.Tags, hostTags)
```

NIT: to avoid calling `GetHostTags` twice. 
(same for line 109 above)